src/rprocessing/RLangPApplet.java

```diff
@@ -324,6 +324,26 @@ private boolean isMixMode() {
   }
 
   /**
+   *
```

Comment:
Could you put these two functions in BuiltinApplet? I think the built-in function wrapper should be there